src/Projections/ProjectionsClient.php

```diff
@@ -329,6 +331,27 @@ public function getStatus(
             $userCredentials,
             HttpStatusCode::OK
         );
+
+        $promise->onResolve(function (?Throwable $exception, ?string $body) use ($deferred): void {
+            if ($exception) {
+                $deferred->fail($exception);
+
+                return;
+            }
+
```

Comment:
```
if (null === $body) {
    $deferred->resolve(new UnexpectedValueException('No content received');

    return;
}

Reply:
I added the same [here](https://github.com/prooph/event-store-client/pull/137) for the other method